provider/consul_catalog.go

```diff
@@ -16,6 +16,8 @@ import (
 const (
 	// DefaultWatchWaitTime is the duration to wait when polling consul
 	DefaultWatchWaitTime = 15 * time.Second
+	// ConsulCatalogTagPrefix is a prefix for additional service/node configurations
+	ConsulCatalogTagPrefix = "traefik."
```

Comment:
I would instead add a `Prefix` field in struct `ConsulCatalog`, like in https://github.com/containous/traefik/blob/master/provider/kv.go#L25.